mappings/net/minecraft/entity/ai/pathing/PathingCoordinator.mapping

```diff
@@ -20,6 +28,17 @@ CLASS none/vx
 		ARG 5 z
 		ARG 6 volume
 		ARG 7 pitch
+	METHOD a didBoundingBoxChange (Lnone/aiw;Lnone/cn;Lnone/asm;Lnone/asm;)Z
```

Comment:
Dirty implies that it needs an update. In this case we do not update the bounding box, we just change the mob's path to account for the changed bounding box.